cloudant.go

```diff
@@ -53,12 +53,12 @@ type Index struct {
 }
 
 // NewClient ...
-func NewClient(username string, password string) (*Client, error) {
-	auth := couchdb.BasicAuth(username, password)
+func NewClient(username string, apikey string, password string) (*Client, error) {
+	auth := couchdb.BasicAuth(apikey, password)
```

Comment:
this is the part I don't understand... should the apikey be the same as username? Did you take a look at the coudb BasicAuth function looks like?  Thank you.